tests/e2e/scale/test_pods_are_not_oomkilled_while_running_ios.py

```diff
@@ -0,0 +1,130 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import ocp, constants, defaults
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.framework.testlib import E2ETest, scale
+from tests.helpers import (
+    default_storage_class,
+    validate_pod_oomkilled
+)
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs.resources.storage_cluster import get_osd_size
+
+log = logging.getLogger(__name__)
+
+
+def validate_pods_are_running_and_not_restarted(
```

Comment:
Why not add this function in helpers ?

Reply:
This function is more specific to tests so defined here. If required will move to helpers

Reply:
I agree with @ramkiperiy. This function could be used for other things, like in cluster health validation and leftovers check

Reply:
moved to helpers.py